CONTRIBUTING.md

```diff
@@ -37,7 +41,8 @@ productivity and inter-personal interactions.
 
 ## Contributor license agreements
 
-We'd love to accept your contributions! But before we can take them, you will have
+We'd love to accept your contributions! But before we can take them, you will
+have 
 to fill out the [Google CLA](https://cla.developers.google.com).
```

Comment:
Seems that there is some reflowing of test, but didn't happen here. Granted it's not going to show differently, so only a comment.